data/json/items/armor.json

```diff
@@ -7399,6 +7399,7 @@
     "weight": 30,
     "volume": 0,
     "price": 590000,
+    "price_postapoc": 59000,
```

Comment:
Both these prices are very still high - almost nobody will be paying much for jewelery, because it really isn't of that much use.

Reply:
Currently they do; whether they *should* is a different matter. None (most?) of the jewelry does not has post-apoc prices, so all are bought/sold at market prices.

My original goal with this PR is just to address the outliers most likely to be abused when trading with NPCs. Reassessing the value of jewelry as a whole for the setting is of a much larger scope, and probably warrants its own issue and a more in-depth discussion to determine what value they'd have in a post-apoc setting.

Reply:
If you're setting a post-apocalyptic price, it NEEDS to be a good price, because other people are going to look for an example to base their own numbers on, and leaving this massively inflated based on the pre-apocalyptic value will lead to other people inflating their prices too.